### ISSUE TITLE
Website Build from zfjournal_source Does Not Push to zfjournal

### ISSUE THREAD

**Issue by akirarorschach** [Zombie-Fridge-Journal/zfjournal_source#1]
The Github Actions push to the external repository `Zombie-Fridge-Journal/zfjournal` fails with the following error: 

Error: Action failed with "The generated GITHUB_TOKEN (github_token) does not support to push to an external repository.

Here is the link to the error: [https://github.com/Zombie-Fridge-Journal/zfjournal_source/runs/3479726805?check_suite_focus=true](url)

**Comment by akirarorschach** [comment]
Alright, so the problem is explained here: [https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-deploy-to-external-repository-external_repository](url). 

I am going to have to update it to use a personal access token, as `GITHUB_TOKEN` as no authorisation to access external repositories.

**Comment by akirarorschach** [comment]
@Zombie-Fridge-Journal/website-administration This issue has been resolved. 

**POSTMORTEM REPORT**

The `GITHUB_TOKEN`, which was automatically generated, did not have authorisation to access external repositories, so the page build could not be pushed into `Zombie-Fridge-Journal/zfjournal`. 

Following the directions in [https://github.com/peaceiris/actions-gh-pages/blob/main/README.md](url), I generated a public/private RSA key pair using the Command Prompt. I added the public key to `Zombie-Fridge-Journal/zfjournal` with write access and added the private key to `Zombie-Fridge-Journal/zfjournal_source`. I then updated `gh-pages.yml`, the Actions workflow document, with the private key. The page build was then successfully pushed to `Zombie-Fridge-Journal/zfjournal`.
